### ISSUE TITLE
Notification from Google Play

### ISSUE THREAD

**Issue by feivur** [WindSekirun/RxSocialLogin#59]
I received notification from Google Play:
> Your application uses a WebView component that is vulnerable to cross-scripting. Detailed information can be found in this article.
> Vulnerable classes:
> `com.kakao.auth.authorization.authcode.KakaoWebViewActivity-> initUi`
> Fix the problem before: 03/18/2020

`gradlew client-v3:dependencies`
shows that this class from RxSocialLogin library:
```
+--- com.github.WindSekirun:RxSocialLogin:1.2.5.3-androidx
|    ...
|    +--- com.kakao.sdk:usermgmt:1.14.0
|    |    \--- com.kakao.sdk:auth:1.14.0
|    |         +--- com.kakao.sdk:network:1.14.0
|    |         |    \--- com.kakao.sdk:util:1.14.0
|    |         \--- androidx.fragment:fragment:1.0.0 -> 1.1.0 (*)
|    ...
```

**Comment by WindSekirun** [comment]
It seems to be they (kakao corp) fix that problem in 1.15.1 (refer https://github.com/JeffGuKang/react-native-kakao-login/issues/6). I'll launch 1.2.5.4-androidx.

**Comment by WindSekirun** [comment]
Released.
```
com.github.WindSekirun:RxSocialLogin:1.2.5.4
com.github.WindSekirun:RxSocialLogin:1.2.5.4-androidx
```
